### PR TITLE
Update prepare-livestream.md

### PR DESCRIPTION
**Pull request by AlbaRoza**
Added a few details to the 'Communication' section.

-----
[View rendered activities/live-streaming/prepare-live-stream.md](https://github.com/publiccodenet/about/blob/AlbaRoza-patch-2/activities/live-streaming/prepare-live-stream.md)

### CODE REVIEW

**Comment by Ainali** [comment]
This is a good addition of details, thanks!

File: activities/live-streaming/prepare-live-stream.md [hunk 1]
```diff
@@ -28,8 +28,8 @@ When setting up a new [live stream](index.md) for 'Let's talk about public code'
 
 Make sure these are ready at least 3 weeks in advance:
 
-- Text messages for social media
-- Images and graphs that can be used in the communication
+- Text messages for social media. Use Planable for the scheduling of those.
```
Comment (Ainali) [inline activities/live-streaming/prepare-live-stream.md:31]: When this ends with a full stop, should all bullet points have full stop? I am unsure. Thoughts @ElenaFdR

Comment (Ainali) [inline activities/live-streaming/prepare-live-stream.md]:
```suggestion
- Text messages for social media. Use Planable for the scheduling of those
```

**Review by Ainali — CHANGES_REQUESTED** [review]

**Review by AlbaRoza — COMMENTED** [review]
Nice add!

**Review by Ainali — APPROVED** [review]

**Pull request merged**